common/script/script-api/src/main/java/org/thingsboard/script/api/tbel/TbUtils.java

```diff
@@ -1437,6 +1446,22 @@ public static int parseBinaryArrayToInt(byte[] bytesValue, int offset, int lengt
         return result;
     }
 
+    public static boolean isInsidePolygon(double latitude, double longitude, String perimeter) {
```

Comment:
I know there is a GeoUtilTest, but please add some primitive test for newly added TBEL functions.